### ISSUE TITLE
Option to simply get raw text preview?

### ISSUE THREAD

**Issue by plpxsk** [sbarex/QLMarkdown#107]
Thanks for this app!

Sometimes, I just want to quick look the markdown files as raw or plain text, without any formatting.

I use this app because natively, macOS can't seem to quick look markdown files as plain text

Using this app, is it possible to simply quick look markdown files as raw or plain text?

**Comment by sbarex** [comment]
Yes, on the `Advanced options`, check the `Render source code` checkbox.
